logger/logger.go

```diff
@@ -416,15 +416,28 @@ func HTTPLogger(inner http.Handler, name string) http.Handler {
 	})
 }
 
+// IsSensitive checks if the given key exists in the list of bad words (sensitive info)
+func IsSensitive(key string) bool {
+	// TODO: Add more sensitive words (lower-case) to this list
+	badWords := map[string]bool{
+		"x-auth-token": true,
+		"username":     true,
+		"user":         true,
+		"password":     true,
+		"passwd":       true,
+		"secret":       true,
+		"token":        true,
+		"accesskey":    true,
```

Comment:
We should include 'accessSecret' as well?

Reply:
Once I go back to the substring check, adding "accessSecret" will not be necessary.